guard/src/lib.rs

```diff
@@ -15,6 +15,15 @@ use crate::commands::test::Test;
 use crate::commands::validate::{OutputFormatType, ShowSummaryType, Validate};
 use crate::commands::Executable;
 pub use crate::rules::errors::Error;
+
+// The following imports are needed for wasm-pack to work
+// When executing npm run build or npm run build:wasm-pack
+#[allow(unused_imports)]
```

Comment:
nit: this is misleading, I think we should use the cfg macro here 

```
#[cfg(wasm)]

Reply:
Nice - thanks! Updated.